ApptentiveFeedback/ApptentiveFeedback/ApptentiveFeedback.nuspec

```diff
@@ -0,0 +1,29 @@
+<package>
+  <metadata>
+    <id>Apptentive.Android</id>
+    <title>Apptentive SDK for Xamarin Android</title>
+    <version>6.0.0</version>
+    <authors>Apptentive</authors>
+    <owners>Apptentive</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>C# bindings for Apptentive SDK</description>
+    <copyright>Copyright © Apptentive</copyright>
+    <projectUrl>https://github.com/apptentive/apptentive-xamarin-android</projectUrl>
+    <licenseUrl>https://raw.githubusercontent.com/apptentive/apptentive-xamarin-android/master/License.txt</licenseUrl>
+    <iconUrl>https://raw.githubusercontent.com/apptentive/apptentive-xamarin-android/master/icons/apptentive_128x128.png</iconUrl>
+    <dependencies>
+      <dependency id="ApptentiveCore" version="6.0.0" />
```

Comment:
Version should be 6.0.4

Reply:
Should't it be 6.0.0? Because its referring to the the Xamarin plugin dependency.

Reply:
Ohh it is referring to Xamarin... Got it!